### PR TITLE
Validate contact form

### PR DESCRIPTION
**Pull request by erickma1**
Implement validation on the contact form to achieve the following

> The content of the email field has to be in lowercase.

The following interactions were also implemented

> When the user submits the form, a check is performed to check if it is in lowercase.
> If the validation is OK, the form is sent.
> If the validation is not OK, an error message is sent and the form is not summited

### CODE REVIEW

**Review by TEMESGEN-G25 — COMMENTED** [review]
## :red_circle: INVALID PULL REQUEST :red_circle: 

## Hi @erickma1 :wave:, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that doesn't have your partner's commits. Since this is a pair-programming project I can't proceed with this review. If the student success team allowed you to work solo on this project, kindly, share a screenshot of the email you've received from them at the PR description.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by erickma1** [comment]
Please my partner made a contribution in the circled portion of the image below.
This is a link to his contribution as well.
https://github.com/erickma1/Module1_Portfolio_Setup/commit/617612f8c47f49c96a8037f0b6e9e2a2b1fd5b69

cephas_contribution_validation

File: js/mobile_menu.js [hunk 1]
```diff
@@ -201,6 +201,31 @@ function closed() {
...
+  const res = email.match(/[A-Z]/g);
+  const btn = document.querySelector('form #form_btn');
+  const container = document.querySelector('form #form-btn-container');
+  const child = document.createElement('h2');
+  let msg = '';
+
+  if (res) {
+    msg = 'Error: Please use only lower case for the email address';
+    event.preventDefault();
+  } else return;
```
Comment (abdulhamiid) [inline js/mobile_menu.js:215]: Requirement screenshot
image

According to the project requirements, when we create the error message, we should stick to the design guidelines (e.g., font, colors, margins) of [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1). 🤗

#### Example, (I'm referring to the `error` styling, not the error message) 👍
image

Your design
image

Please kindly adjust your design. 💪

**Review by abdulhamiid — CHANGES_REQUESTED** [review]
# Status: Required Changes :recycle:
Hi Team 👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there! :running_man: :runner: 
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights 🌟

- ✔️ Descriptions are descriptive
- ✔️ Linters checks are passing
- ✔️ Use GitHub Flow
- ✔️ Meaningful commit messages.

## Required Changes ♻️
_Please kindly check the comments under the review._

## Optional suggestions 🤷‍♂️
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

## Important Notes 📝
- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by erickma1** [comment]
Add font format for error message to match Figma design

**Review by 1GORDON — APPROVED** [review]
# Status: APPROVED 🟢 

Hi @erickma1 ✋

Your project is complete!  There is nothing else to say other than... it's time to merge it: ship it:
Congratulations! :tada: :clap: :clap: :100:

## Highlights!
- Followed GitHub flow ✔️
- Good commit messages ✔️
- Linters are passing ✔️
- Descriptive Pull request ✔️
- Professional readme ✔️

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend considering them as they can improve your code._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@1GORDON` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**